apps/condo/domains/acquiring/schema/MultiPayment.js

```diff
@@ -270,8 +270,11 @@ const MultiPayment = new GQLListSchema('MultiPayment', {
                 })
                 const oldStatus = existingItem.status
                 const newStatus = get(resolvedData, 'status', oldStatus)
-                if (!MULTIPAYMENT_TRANSITIONS[oldStatus].includes(newStatus)) {
-                    return addValidationError(`${MULTIPAYMENT_NOT_ALLOWED_TRANSITION} Cannot move from "${oldStatus}" status to "${newStatus}"`)
+                // we can not use resolvedData.hasOwnProperty('status') check here as adminUi sends status if it is not changed
+                if (oldStatus !== newStatus) {
+                    if (!MULTIPAYMENT_TRANSITIONS[oldStatus].includes(newStatus)) {
+                        return addValidationError(`${MULTIPAYMENT_NOT_ALLOWED_TRANSITION} Cannot move from "${oldStatus}" status to "${newStatus}"`)
```

Comment:
Is it possible to write a test to check this error?

Reply:
There was such test already
"Cannot change statuses if it\'s transition is not specified"